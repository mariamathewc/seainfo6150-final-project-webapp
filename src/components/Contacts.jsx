import React from "react";
import styles from "./Contacts.module.css";
import summer from "../images/snow.jpg"

const Contacts = (props) => {




    return (
        <div>
            <div className={styles.imageContainer}>
                <img className={styles.iconDetails} src={summer} alt="summer camp" />
                </div>

            <div className={styles.container}>
                <address >
                    < div className={styles.address}>
                  
                    <h3>Address:</h3>
                    <p> 1700 N 1st St</p>
                    <p> San Jose</p>
                    <p> California</p>
                    <p> 95112</p>
                    <br />
                    <br />
                    <div><h3>Email Address:</h3> mathew.m@husky.neu.edu</div>
                    <br />
                    <br />
                        <div><h3>Contact No:</h3> 510 309 2054</div>
                       
                </div>
            </address>
          
        </div>
</div>



    );


}

export default Contacts