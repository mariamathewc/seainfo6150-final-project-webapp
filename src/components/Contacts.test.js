import React from "react";
import Contacts from "./Contacts.jsx";
import { render } from "@testing-library/react";
import { BrowserRouter } from 'react-router-dom';



describe("Home tests", () => {
	it("renders correctly", () => {
		const { container } = render(<BrowserRouter><Contacts /></BrowserRouter>);
		expect(container).toMatchSnapshot();
	});
});