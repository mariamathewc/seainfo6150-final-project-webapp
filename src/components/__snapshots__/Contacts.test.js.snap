// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home tests renders correctly 1`] = `
<div>
  <div>
    <div
      class="imageContainer"
    >
      <img
        alt="summer camp"
        class="iconDetails"
        src="snow.jpg"
      />
    </div>
    <div
      class="container"
    >
      <address>
        <div
          class="address"
        >
          <h3>
            Address:
          </h3>
          <p>
             1700 N 1st St
          </p>
          <p>
             San Jose
          </p>
          <p>
             California
          </p>
          <p>
             95112
          </p>
          <br />
          <br />
          <div>
            <h3>
              Email Address:
            </h3>
             mathew.m@husky.neu.edu
          </div>
          <br />
          <br />
          <div>
            <h3>
              Contact No:
            </h3>
             510 309 2054
          </div>
        </div>
      </address>
    </div>
  </div>
</div>
`;
